docs/modules/ROOT/pages/how-tos/upgrade-1.1.1-1.7.x.adoc

```diff
@@ -0,0 +1,9 @@
+= Upgrade from 1.1.x to 1.2.x
+
+This guide describes the steps to perform an upgrade of the component from version 1.1.x to 1.2.x.
+
+== Step-by-step guide
+
+Component version 1.2.x installs Crossplane version 1.7.1. Since Crossplane Version 1.3, all CRDs are being managed by the Crossplane Operator and no longer by the Helm Chart. An automated PreSync Hook during migration will ensure that CRDs aren't deleted.
```

Comment:
Please keep all documentation to a single sentence per line. This makes reviewing it and suggesting changes much easier.

```suggestion
Component version v2.0.0 installs Crossplane version 1.7.1.
Since Crossplane Version 1.3, all CRDs are being managed by the Crossplane Operator and no longer by the Helm Chart.
An automated PreSync Hook during migration will ensure that CRDs aren't deleted.
```

Reply:
I forget about this rule all the time